test/perf/perf_test.go

```diff
@@ -75,7 +75,7 @@ func TestBenchmarkCopyingLargeBundleThatContainsImagesMostlyOnDockerHub(t *testi
 		maxTimeCopyShouldTake = 8 * time.Minute.Nanoseconds()
 	}
 
-	require.Greaterf(t, maxTimeCopyShouldTake, actualTimeTaken, "copying a large bundle took too long. Expected it to take maximum [%v] but it took [%v]", time.Duration(maxTimeCopyShouldTake), time.Duration(actualTimeTaken))
+	require.Greaterf(t, maxTimeCopyShouldTake, actualTimeTaken, fmt.Sprintf("copying a large bundle took too long. Expected it to take maximum [%v] but it took [%v]", time.Duration(maxTimeCopyShouldTake), time.Duration(actualTimeTaken)))
```

Comment:
Check if this is needed, if it is please have a separated commit for it

Reply:
Yeah we need it. The GreaterF function has a bug around printing a msg with args. It ends up looking like this otherwise:
```
Messages:   	[copying a large bundle took too long. Expected it to take maximum [%v] but it took [%v] 123ns 321ns]
```

I'll create a separate commit for this.